src/Common/WorkloadInstall.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+static class WorkloadInstall
```

Comment:
Can we have a more descriptive name here? Maybe `WorkloadInstallConfiguration`?

Reply:
> Also, is there a reason this class isn't in a namespace? Does that make it easier to use as shared common code?

No. I've renamed it to `Microsoft.DotNet.Workloads.Workload.WorkloadFileBasedInstall`.